makefile.include

```diff
@@ -2,6 +2,13 @@
 # Include makefile for libtomcrypt
 #
 
+# The version
```

Comment:
I used to use MIN + MAJ in `makefile.unix` for creating shared libraries which I later removed. So leaving them out is fine.